.github/workflows/conversion_script.yml

```diff
@@ -0,0 +1,63 @@
+name: Generate Conversion Script
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '**/cellxgene_schema_cli/cellxgene_schema/ontology_files/*.csv.gz'
+      - '**/cellxgene_schema_cli/cellxgene_schema/ontology_files/*_diff.txt'
+      - '**/cellxgene_schema_cli/cellxgene_schema/ontology_files/all_ontology.json.gz'
+
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  create-conversion-script:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.11
+      - name: Python cache
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements*.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: Install dependencies
+        run: |
+          pip install -r migration_assistant/requirements.txt
+      - name: Run Conversion Script Generator
+        run: |
+          make generate-conversion-script
+      - name: Get Version
+        id: get_version
+        run: |
+          echo "old_version=$(make show-current-version)" >> $GITHUB_ENV
+      - name: setup git
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+      - name: Bump Version
+        run: |
+          pip install bumpversion
+          pip install wheel
+          bumpversion --config-file .bumpversion.cfg patch --allow-dirty
+          bumpversion --config-file .bumpversion.cfg prerel --allow-dirty --tag
+          echo "new_version=$(make show-current-version)" >> $GITHUB_ENV
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
+        with:
+          commit-message: 'AUTO: update migrate.py schema_version ${{ env.old_version }}->${{ env.new_version }}'
+          title: 'AUTO: update migrate.py schema_version ${{ env.old_version }}->${{ env.new_version }}'
+          body: 'This is an automated PR to update migrate.py from schema_version ${{ env.old_version }}->${{ env.new_version }}.'
+          branch: auto/update-convert-py-to-${{ env.new_version }}
```

Comment:
nit: update-migrate-py

Reply:
actually--can you remove this file from this PR so we can consolidate changes to this file to [this PR?](https://github.com/chanzuckerberg/single-cell-curation/pull/467/files#diff-bc6ed18ad505a5b48b177afd813bc6afa2a502bc849d939f87b5edeee2259d87)